src/me/mrCookieSlime/HotbarPets/HotbarPet.java

```diff
@@ -8,19 +8,20 @@
 import org.bukkit.inventory.ItemStack;
 
 public class HotbarPet extends SlimefunItem {
-	
+
     private static Category category;
-    ItemStack food;
 
     static {
         try {
-            category = new Category(new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNjIxNjY4ZWY3Y2I3OWRkOWMyMmNlM2QxZjNmNGNiNmUyNTU5ODkzYjZkZjRhNDY5NTE0ZTY2N2MxNmFhNCJ9fX0="), "§dHotbar Pets", new String[]{"", "&a> Click to open"}));
+            category = new Category(new CustomItem(CustomSkull.getItem("eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvNjIxNjY4ZWY3Y2I3OWRkOWMyMmNlM2QxZjNmNGNiNmUyNTU5ODkzYjZkZjRhNDY5NTE0ZTY2N2MxNmFhNCJ9fX0="), "§dHotbar Pets", "", "&a> Click to open"));
```

Comment:
I also just noticed:
That § should be an &

Reply:
There we go